### ISSUE TITLE
Organize questions into categories

### ISSUE THREAD

**Issue by jimjshields** [thomasballinger/pythonquiz#5]
Allow categorization of questions by concept - use the mindmap we made as a guide. 
Abstract this so that anyone could create categorizations.

**Comment by thomasballinger** [comment]
Ooo this is cool. Maybe allow tagging or something? Right now the only way to collect questions for a quiz is to use a different file because it's a global questions list.

However, I do sort of like the direct correspondence between source files and resulting quizzes - building another layer for constructing the quizzes feels unnecessary to me right now - so maybe just labels or tags for now?

**Comment by jimjshields** [comment]
Yeah, the way I was thinking of implementing was exactly the same way as explanations - you add a decorator through which you pass a label/tag/category and it adds it as an attribute to the question. Is that a fair way to go about it?

It's also not a necessary layer - like explanations it'd be optional but would render in some way on the front-end (though I'm not sure how through the terminal).

**Comment by thomasballinger** [comment]
No totally this sounds great - my mind took off and started thinking about
how these might be used, and I agree it should basically be meta info
shipped off to the frontend where we can do something with it. (Not
additional preprocessing in the backend)
